test/resources/model_validation_results/model-microsoft-good/spec.yaml

```diff
@@ -13,5 +13,6 @@ properties:
   SHA: 9da301148150e37e533abef672062fa49f6bda4f
   inference-min-sku-spec: 4|0|14|28
   inference-recommended-sku: Standard_DS3_v2
+  SharedComputeCapacityEnabled: true
```

Comment:
Thanks for the unit tests! Consider adding a bad example too.

Reply:
Yes there, are 2 bad examples added previously. Not part of this PR though.